### ISSUE TITLE
Using the new layout helpers does not work

### ISSUE THREAD

**Issue by hfjallemark** [mhanberg/temple#85]
The new layout helpers that were released in Phoenix 1.5.0 does not work well with Temple 0.5.

E.g. using something like:

```
# base.html.eex
...
<%= @inner_content %>
...

# app.html.eex
render_layout MyAppWeb.LayoutView, "base.html", assigns do
  ...
  @inner_content
  ...
end
```

**Comment by mhanberg** [comment]
Hopefully this will be fixed by 0.6.

**Comment by mhanberg** [comment]
This should work now I believe.
